metadata.yaml

```diff
@@ -1,9 +1,10 @@
-name: alejdg-jenkins-agent-k8s
+name: jenkins-agent
```

Comment:
The name should be `jenkins-agent-k8s` as this will replace https://charmhub.io/jenkins-agent-k8s.

Reply:
Thanks, fixed